packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -74,69 +75,81 @@ export default {
         altText: {
             type: String,
             default: 'Media Element Image'
-        },
-        stackWhenNarrow: {
-            type: Boolean,
-            default: false
         }
     },
+
     data () {
         return {
             componentName: 'MediaElement'
         };
     },
+
     computed: {
         /**
          * Returns the class to modify the alignment of the content based on the prop contentAlign
          * @returns {*}
          */
         contentAlignClass () {
-            switch (this.contentAlign) {
-                case ALIGN.LEFT:
-                    return this.$style['c-mediaElement-content--left'];
-                case ALIGN.RIGHT:
-                    return this.$style['c-mediaElement-content--right'];
-                case ALIGN.CENTER:
-                    return this.$style['c-mediaElement-content--center'];
-                default:
-                    return this.$style['c-mediaElement-content--left'];
-            }
+            const alignment = Object.keys(ALIGN)
```

Comment:
Sometimes I prefer `switch` statements, they appear to read a bit easier to me.

Reply:
I can agree to a point, I agree when we have a limited set of cases they can indeed be easier to read, but in this case I am thinking ahead for what I would like to do next which is to allow alignments on top of left,center and right. This is when switch statements begin to smell a bit for me, when your writing essentially the same thing out multiple times and you know that additional cases could be added in the future. This is where I think a simple object lookup via key works best as I just add more options into the object if I want in this case additional alignments.